samples/snippets/src/main/java/com/example/bigquerystorage/WriteToDefaultStream.java

```diff
@@ -39,27 +37,31 @@ public static void runWriteToDefaultStream()
     String projectId = "MY_PROJECT_ID";
     String datasetName = "MY_DATASET_NAME";
     String tableName = "MY_TABLE_NAME";
-
-    writeToDefaultStream(projectId, datasetName, tableName);
+    TableFieldSchema strField =
```

Comment:
I would suggest keep the sample the way it is. Move BQV2ToBQStorageConverter.java into sample as a lib, and the sample would still do a GetTable(), use BQV2ToBQStorageConverter to convert the schema and then use JsonWriter. It doesn't need to block this PR.

Reply:
Sure. Will make this change in a separate PR.